packages/stream2peer-landing/src/components/Hero.tsx

```diff
@@ -6,7 +6,7 @@ import { GitHubLogoIcon } from "@radix-ui/react-icons";
 export const Hero = () => {
 
   return (
-    <section className="container grid lg:grid-cols-2 place-items-center py-20 md:py-32 gap-10">
+    <section className="container overflow-hidden grid lg:grid-cols-2 place-items-center py-20 md:py-32 gap-10">
```

Comment:
Why not `overflow-x-hidden`?

Reply:
overflow-x-hidden causes the overflow to shift to the y axis, so just overflow-hidden gives no room for both x or y overflow

Reply:
Oya now! Omo ologo 🫡